src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Embedded.cs

```diff
@@ -185,7 +185,7 @@ public static void Main()
     }
 }";
 
-            CompileAndVerify(code, verify: false, expectedOutput: "3");
```

Comment:
So we're moving from skipping verify to asserting that it passes? I don't see the reasoning.

Reply:
I changed CompilationVerifier.cs to Assert.Throws when Verification.Fails. That flags some places that actually do verify. 
I didn't remove Verification.Passes yet, but we could (since that's the default).

---
In reply to: [150988118](https://github.com/dotnet/roslyn/pull/23153#discussion_r150988118) [](ancestors = 150988118)